tests/data_context/test_data_context_state_management.py

```diff
@@ -226,246 +218,6 @@ def test_update_project_config(
     assert context.progress_bars["globally"] is True
 
 
-@pytest.mark.unit
-def test_add_datasource_with_existing_datasource(
```

Comment:
we're going to remove datasource mgmt from context, so no use in reworking these tests.